### ISSUE TITLE
[Design System] Migrate MenuDropdown to ChakraUI

### ISSUE THREAD

**Issue by seaerchin** [isomerpages/isomercms-frontend#862]
**work required**
`MenuDropdown` is functionally similar to `Dropdown`, with the exception that the items are fixed and require an icon. Migration of this component to design system could be done in a few ways, outlined below (eng picking up ticket to decide) 

1. remove and use chakraUI's dropdown (similar to #860)
2. refactor `MenuItem` to have 3 **stylistic** variants (eg: `<MenuItem variant = "delete" />`
3. refactor `MenuItem` to be 3 different components (`DeleteItem/MoveItem/EditItem`)

Do also take note that by adopting ChakraUI's component, the `showDropdown && <MenuDropdown />` pattern can (and should) also be removed

owing to the similarities between this component and #860, it is best if this is deferred till after #860 is done.
